pkgs/development/tools/continuous-integration/gitea-actions-runner/default.nix

```diff
@@ -1,18 +1,21 @@
-{ lib, fetchFromGitea, buildGoModule }:
+{ lib
+, fetchFromGitea
+, buildGoModule
+}:
 
 buildGoModule rec {
   pname = "gitea-actions-runner";
-  version = "unstable-2023-03-18";
+  version = "0.1.2";
 
   src = fetchFromGitea {
     domain = "gitea.com";
     owner = "gitea";
     repo = "act_runner";
-    rev = "9eb8b08a69e8b1c699c9c07a06c1ff8e5f6ad0fe";
-    sha256 = "sha256-B8vD+86X8cqZhPmDmEjHgSsq3TdJuCf9h3XgdXC7hQY=";
+    rev = "v${version}";
+    hash = "sha256-0wLBFZdFaGTselRZIJ809wHQFhAjt4sdmZp12LQOB9I=";
   };
 
-  vendorSha256 = "sha256-K/d/ip8icc+rjTmajsGxw5aij1VMW6wJJu4LCkKqaVQ=";
+  vendorHash = "sha256-GBqt5qSM2mDAS7klWdJiC6t5HaVKaP0PjecYnHZBzrc=";
 
   ldflags = [
     "-s"
```

Comment:
Can you update the ldflag to use `-X gitea.com/gitea/act_runner/internal/pkg/ver.version=${version}` instead?

Reply:
Applied, thanks!